src/Neo/SmartContract/Native/StdLib.cs

```diff
@@ -222,5 +223,31 @@ private static string[] StringSplit([MaxLength(MaxInputLength)] string str, stri
             StringSplitOptions options = removeEmptyEntries ? StringSplitOptions.RemoveEmptyEntries : StringSplitOptions.None;
             return str.Split(separator, options);
         }
+
+        [ContractMethod(CpuFee = 1 << 8)]
+        private static int StringCharLength([MaxLength(MaxInputLength)] string str)
+        {
+            // return the length of the string in characters
+            // it should return 2 for "🦆" and 1 for "ã"
```

Comment:
Do we need it? It's tied to .NET-specific definition of `Char` and UTF-16 which is very niche. At the moment I just don't know how it could be implemented in Go. [`utf8.RuneCountInString`](https://pkg.go.dev/unicode/utf8@go1.20.2#RuneCountInString) works fine or `StringElementLength`, but `StringCharLength` looks very much C#/.NET specific and not very useful at the first glance.

Reply:
Will remove this one then.

Reply:
@roman-khimov updated, may you please review again

Reply:
Looks OK, just one simple method.